### ISSUE TITLE
wallet-cli and wallet-rpc fail during refreshing

### ISSUE THREAD

**Issue by fuwa0529** [wownero/wownero#96]
When restoring one of my wallet from seed, wallet-cli will silently fail, but wallet-rpc will give the following error:

```
2018-09-28 07:30:59.441     7f99f4f55c00        WARN    global  src/wallet/wallet2.cpp:1513     **********************************************************************
2018-09-28 07:30:59.441     7f99f4f55c00        WARN    global  src/wallet/wallet2.cpp:1514     Consistency failure in amounts received
2018-09-28 07:30:59.441     7f99f4f55c00        WARN    global  src/wallet/wallet2.cpp:1515     Check transaction <9e6a56a07ea3af62fef24db4926dcdd35c5b10fbf25fcebea13d1317383267b4>
2018-09-28 07:30:59.441     7f99f4f55c00        WARN    global  src/wallet/wallet2.cpp:1516     **********************************************************************
```

This is on wowario/wownero 's fork, 458155e at the monero branch.

**Comment by wowario** [comment]
Issue fixed with #93

**Comment by fuwa0529** [comment]
such fix!
